multinode_integration_tests/tests/verify_bill_payment.rs

```diff
@@ -183,15 +186,15 @@ fn payments_were_adjusted_due_to_insufficient_balances() {
             .unwrap()
         };
         let transaction_fee_margin = PurePercentage::try_from(15).unwrap();
-        transaction_fee_margin.add_percent_to(gas_limit_dev_chain * gas_price_major)
+        transaction_fee_margin.increase_by_percent_for(gas_limit_dev_chain * gas_price_major)
```

Comment:
You don't need to keep the word `_for` in the functions's name. Please eliminate it.